src/main/java/com/squareup/phrase/Phrase.java

```diff
@@ -214,6 +230,16 @@ public CharSequence format() {
     return formatted;
   }
 
+  /**
+   * This can be useful for unit tests that don't want to use {@link SpannableStringBuilder}
+   * which can cause a stub exception or be no-op.
+   * Default is <code>true</code>.
+   */
+  @SuppressWarnings("UnusedDeclaration") // Public API.
+  public void setSpanSupportEnabled(boolean spanSupportedEnabled) {
```

Comment:
Have you considered just making this variable final, and having new factory methods for it?  There'd be a lot of method proliferation I guess.

I was thinking of something like `Phrase.stubSpan().from(...)` or `Phrase.stubSpanFrom(...)` or something.  That way you just have a small amount of code to add across a lot of tests.

I'd just like someone to avoid running into this method in their normal Phrase usage.